numpy/lib/tests/test_recfunctions.py

```diff
@@ -450,8 +450,8 @@ def test_flatten_wflexible(self):
         test = merge_arrays((x, w), flatten=False)
```

Comment:
The test above this one is the interesting one - before it would _not_ flatten 0-field structs, but these also weren't tested